src/main/java/org/folio/rest/impl/HoldingsStorageAPI.java

```diff
@@ -36,7 +45,11 @@
   // Has to be lowercase because raml-module-builder uses case sensitive
   // lower case headers
   private static final String TENANT_HEADER = "x-okapi-tenant";
+  private static final String WHERE_CLAUSE = "WHERE id = '%s'";
   public static final String HOLDINGS_RECORD_TABLE = "holdings_record";
+  public static final String ITEM_TABLE = "item";
+  public static final int NO_OFFSET = 0;
```

Comment:
Is this used anywhere? I don't see anything in the commit, but it is public, so maybe something else is using it. If it is being used, I am not sure this is the right place for this constant.